### PR TITLE
Tao 3381/jwt signatures

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: model/ontology/openid.rdf [hunk 1]
```diff
@@ -5,6 +5,7 @@
         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
         xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
         xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
```
Comment (jbout) [inline model/ontology/openid.rdf]: duplicate NS declaration

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -44,8 +44,10 @@ public function update($initialVersion)
             AclProxy::applyRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/generis.rdf#AnonymousRole', ['ext'=>'taoOpenId','mod' => 'Connect', 'act' => 'callback']));
             $this->setVersion('0.0.3');
         }
-        
-        if ($this->isVersion('0.0.3')) {
+
+        $this->skip('0.0.3', '0.1.0');
```
Comment (jbout) [inline scripts/update/Updater.php]: modification of past update scripts should only be done in bug fixes that fix issues with update scripts

**Review by jbout — CHANGES_REQUESTED** [review]
Please also remerge develop

**Comment by zagovorichev** [comment]
@jbout Update

**Review by jbout — APPROVED** [review]

**Pull request merged**